Dockerfile.elixir

```diff
@@ -14,8 +14,10 @@ RUN apk -U add \
     erlang-parsetools=$ERLANG_PACKAGE_VERSION \
     erlang-public-key=$ERLANG_PACKAGE_VERSION \
     erlang-ssl=$ERLANG_PACKAGE_VERSION \
-    erlang-tools=$ERLANG_PACKAGE_VERSION \
-    erlang-syntax-tools=$ERLANG_PACKAGE_VERSION
+    erlang-syntax-tools=$ERLANG_PACKAGE_VERSION \
+    erlang-xmerl=$ERLANG_PACKAGE_VERSION \
+    erlang-mnesia=$ERLANG_PACKAGE_VERSION \
+    erlang-tools=$ERLANG_PACKAGE_VERSION
```

Comment:
I'm terrible for not having done this here, but could you sort the packages alphabetically? It makes it a little easier to work with.